protocol-designer/src/pages/Landing/index.tsx

```diff
@@ -1,27 +1,94 @@
 import * as React from 'react'
-import { NavLink } from 'react-router-dom'
+import { NavLink, useNavigate } from 'react-router-dom'
+import styled from 'styled-components'
+import { useDispatch } from 'react-redux'
+import { useTranslation } from 'react-i18next'
 import {
+  ALIGN_CENTER,
+  COLORS,
+  DIRECTION_COLUMN,
   Flex,
+  LargeButton,
   SPACING,
-  DIRECTION_COLUMN,
-  ALIGN_CENTER,
+  StyledText,
+  TYPOGRAPHY,
 } from '@opentrons/components'
-import { useTranslation } from 'react-i18next'
+import { actions as loadFileActions } from '../../load-file'
+import welcomeImage from '../../images/welcome_page.png'
+import type { ThunkDispatch } from '../../types'
 
 export function Landing(): JSX.Element {
   const { t } = useTranslation('shared')
+  const dispatch: ThunkDispatch<any> = useDispatch()
+  const navigate = useNavigate()
+  const loadFile = (
+    fileChangeEvent: React.ChangeEvent<HTMLInputElement>
+  ): void => {
+    if (window.confirm(t('confirm_import') as string)) {
+      dispatch(loadFileActions.loadProtocolFile(fileChangeEvent))
+      navigate('/overview')
+    }
+  }
 
   return (
-    <Flex flexDirection={DIRECTION_COLUMN} margin={SPACING.spacing24}>
-      {t('create_opentrons_protocol')}
-      <Flex
-        alignItems={ALIGN_CENTER}
-        marginTop={SPACING.spacing24}
-        gridGap={SPACING.spacing16}
+    <Flex
+      backgroundColor={COLORS.grey20}
+      flexDirection={DIRECTION_COLUMN}
+      alignItems={ALIGN_CENTER}
+      paddingTop="14.875rem"
+      height="100vh"
+      width="100%"
+    >
+      <img
+        src={welcomeImage}
+        height="132px"
+        width="548px"
+        aria-label="welcome image"
+      />
+      <StyledText
+        desktopStyle="headingLargeBold"
+        marginTop={SPACING.spacing16}
+        marginBottom={SPACING.spacing16}
+      >
+        {t('welcome')}
+      </StyledText>
+      <StyledText
+        desktopStyle="headingSmallRegular"
+        color={COLORS.grey60}
+        maxWidth="34.25rem"
+        textAlign={TYPOGRAPHY.textAlignCenter}
       >
-        <NavLink to={'/createNew'}>{t('create_new')}</NavLink>
-        {t('import')}
-      </Flex>
+        {t('no-code-solution')}
+      </StyledText>
+      <LargeButton
+        margin={SPACING.spacing32}
```

Comment:
from the screenshot I guess the following would be enough (only top and bottom)?

```suggestion
        marginY={SPACING.spacing32}
```